contrast_api/serializers.py

```diff
@@ -19,6 +19,16 @@ class TrendsOverYearsGraphSerializer(serializers.Serializer):
     series = YearlySeriesSerializer(many=True)
 
 
+class HistogramSerializer(serializers.Serializer):
+    key = serializers.DecimalField(max_digits=10, decimal_places=2)
```

Comment:
מפתיע אותי שה key פה הוא מספר עשרוני אבל ה value הוא מספר שלם

Reply:
because the key is a continuous value (e.g "duration" for example) while the "value" here is the number of experiments which is an integer